frontend/src/components/HomePage.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import Header from '@/components/Header.vue'
+import BetaversionLabel from '@/components/betaversion-label.vue'
+
+const emit = defineEmits(['acceptDisclaimer'])
+
+</script>
+
+<template>
+    <Header />
+    <main>
+      <div class="rvo-progress-tracker">
+      </div>
+      <div class="rvo-form">
+        <div class="flex items-center rvo-layout-column rvo-layout-gap--xl">
+          <div class="flex justify-center rvo-form-intro inline-flex py-5">
+            <h2 class="flex justify-center utrecht-heading-2">Beslisboom AI-verordering</h2>
+            <BetaversionLabel />
+          </div>
+          <div class="rvo-layout-column rvo-layout-gap--md">
+            <div class="utrecht-form-fieldset rvo-form-fieldset">
+              <fieldset
+                class="utrecht-form-fieldset__fieldset utrecht-form-fieldset--html-fieldset"
+              >
+                <p class="utrecht-paragraph">Met deze beslisboom stel je vast of de AI-verordening geldt voor jouw
+                  overheidsorganisatie. En aan welke
+                  vereisten je dan moet voldoen.</p>
+                <ul class="utrecht-unordered-list">
+                  <li class="utrecht-unordered-list__item">Deze beslisboom is een betaversie: de informatie is niet
+                    compleet en er kunnen fouten in staan.
+                  </li>
+                  <li class="utrecht-unordered-list__item">Je bent zelf verantwoordelijk voor de informatie die je
+                    gebruikt.
+                  </li>
+                  <li class="utrecht-unordered-list__item">Overleg de uitkomsten van de beslisboom met een expert.</li>
+                  <li class="utrecht-unordered-list__item">De beslisboom is bedoeld voor overheden die AI-systemen
+                    aanbieden, of gebruiken onder eigen
+                    verantwoordelijkheid.
+                  </li>
+                  <li class="utrecht-unordered-list__item"> Voor vragen of opmerkingen over de beslisboom mail je naar:
+                    <a
+                      class="rvo-link" href="mailto:ai-verordering@minbzk.nl">
+                      ai-verordering@minbzk.nl</a>.
+                  </li>
+                </ul>
```

Comment:
This list results in items being very close to the bullet. I suggest trying this instead:

<ul class="rvo-ul rvo-ul--no-margin rvo-ul--no-padding"\
  <li>Voorbeeld van een unordered list</li>
  <li>Item 2</li>
  <li>Item 3</li>
</ul>

It looks better. Found it here: https://minbzk.github.io/NL-design-system/?path=/story/nl-rvo-design-system_pagina-s-templates-content--default

Reply:
Good one, not just adding it to the class of the ul but also remove the class individually of the list items did the trick, thanks for this tip :)